contracts/2key/singleton-contracts/TwoKeyUpgradableExchange.sol

```diff
@@ -348,6 +354,12 @@ contract TwoKeyUpgradableExchange is Upgradeable, MaintainingPattern {
 
         token.transferFrom(msg.sender, address(this), _twoKeyUnits);
         uint stableCoinsAfter = stableCoinsOnContractBefore - stableCoinUnits;
+
+        if (ERC20(DAI).balanceOf(this) < stableCoinUnits){
+            emit LackOfStableCoinInventory(stableCoinsOnContractBefore, _twoKeyUnits, stableCoinUnits);
```

Comment:
This is not good

Reply:
Will change to:

ERC20(DAI).balanceOf(address(this)) /10**18